api/pkg/db/initializer/initializer.go

```diff
@@ -31,6 +32,13 @@ type Initializer struct {
 	api app.BaseConfig
 }
 
+var (
+	apiserverID       = 100000
```

Comment:
can we give some id which cannot be there in current already deployed hub db - something like zero etc

Reply:
it' s not working with id value zero or negative, can you please suggest any other id ?

Reply:
since we are using uint so negative integers are not allowed, how about we use it's max possible value `4,29,49,67,295` ?